README/tr_TR.md

```diff
@@ -0,0 +1,86 @@
+<div align="right">Language: <a title="English" href="https://github.com/yrccondor/mdx/blob/master/README.md" | <a title="Chinese" href="https://github.com/yrccondor/mdx/blob/master/README/zh_CN.md">Chinese</a> | Turkish</div>
+
+<br>
+
+<p align="center">
+<img src="https://img.flyhigher.top/top.jpg" alt="MDx">
+</p>
+
+<h1 align="center"><a href="https://mdx.flyhigher.top" target="_blank">MDx</a></h1>
+
+> Basit görünüşlü, güçlü.
+
+<p align="center">
+<img alt="Versiyon" src="https://img.shields.io/badge/version-1.8.12-3f51b5.svg?style=flat-square"/>
+<img alt="sürüm" src="https://img.shields.io/badge/dynamic/json.svg?label=build&url=https%3A%2F%2Fbuildmdx.flyhigher.top%2Fbuild.json&query=%24.sta&colorB=44cc11&style=flat-square"/>
+<a href="https://github.com/yrccondor/mdx/releases"><img alt="sürümler" src="https://img.shields.io/github/release/yrccondor/mdx.svg?style=flat-square"/></a>
+<a href="https://flyhigher.top"><img alt="Yazar" src="https://img.shields.io/badge/author-Axton-red.svg?style=flat-square"/></a>
+<img alt="WordPress" src="https://img.shields.io/badge/WordPress-4.4%2B-blue.svg?style=flat-square"/>
+<a href="https://mdx.flyhigher.top"><img alt="İndir" src="https://img.shields.io/badge/download-1.19M-brightgreen.svg?style=flat-square"/></a>
+<a href="https://github.com/yrccondor/mdx/blob/master/LICENSE"><img alt="Lisans" src="https://img.shields.io/badge/license-GPL%20V3.0-orange.svg?style=flat-square"/></a>
+<a href="https://app.fossa.io/projects/git%2Bgithub.com%2Fyrccondor%2Fmdx?ref=badge_shield" alt="FOSSA Status"><img src="https://app.fossa.io/api/projects/git%2Bgithub.com%2Fyrccondor%2Fmdx.svg?type=shield"/></a>
+</p>
+
+
+## İçerikler
+
+- [Tanıtım](#introduction)
```

Comment:
Hashtag link should be in Turkish and be URLEncoded in lower case like this:
```suggestion
- [Tanıtım](#tan%C4%B1t%C4%B1m)
```
Same in other items.